telethon/client/updates.py

```diff
@@ -279,6 +279,30 @@ async def catch_up(self: 'TelegramClient'):
 
     # endregion
 
+    async def fetch_updates(self: 'TelegramClient'):
```

Comment:
I don't want to add this method because the whole update handling needs to be remade and more carefully thought out. I'm also not sure if this would be the implementation I'd go for; if there are thousands of updates, thousands of tasks will be made, and this is not respecting the sequential updates.